cirq/circuits/circuit.py

```diff
@@ -59,11 +59,15 @@ class Circuit:
         all_qubits
         all_operations
         findall_operations
+        findall_operations_between
         findall_operations_until_blocked
         findall_operations_with_gate_type
+        reachable_frontier_from
+        has_measurements
         are_all_matches_terminal
         are_all_measurements_terminal
         unitary
+        final_wavefunction
         apply_unitary_effect_to_state
```

Comment:
Remove `apply_unitary_effect_to_state()`. It is deprecated and users should use `final_wavefunction()` instead.

Reply:
Done